services/web/client/source/class/qxapp/Application.js

```diff
@@ -135,6 +136,37 @@ qx.Class.define("qxapp.Application", {
     __disconnectWebSocket: function() {
       // open web socket
       qxapp.wrapper.WebSocket.getInstance().disconnect();
-    }
+    },
+
+    __preventAutofillBrowserSyles: function() {
+      const stylesheet = qx.ui.style.Stylesheet.getInstance();
+      if (qx.bom.client.Browser.getName() === 'chrome' && qx.bom.client.Browser.getVersion() >= 71) {
+        stylesheet.addRule(
+          'input:-internal-autofill-previewed,' +
+          'input:-internal-autofill-selected,' +
+          'textarea:-internal-autofill-previewed,' +
+          'textarea:-internal-autofill-selected,' +
+          'select:-internal-autofill-previewed,' +
+          'select:-internal-autofill-selected',
+
+          'transition: background-color 0s linear 5000s, color 0s linear 5000s'
+        );
+      }
+      if (qx.bom.client.Engine.getName() === 'webkit') {
```

Comment:
I would use `else if` here

Reply:
I thought the specific Chrome status flags were maybe used for some other internal behavior, but now I think you are right, because if both are applied, the webkit ones get disabled. It's done.